test/integration/package_int_test.go

```diff
@@ -313,29 +314,38 @@ func (suite *PackageIntegrationTestSuite) TestPackage_operation() {
 	username := ts.CreateNewUser()
 	namespace := fmt.Sprintf("%s/%s", username, "python3-pkgtest")
 
-	cp := ts.Spawn("fork", "ActiveState-CLI/Python3", "--org", username, "--name", "python3-pkgtest")
+	cp := ts.Spawn("fork", "ActiveState-CLI/Revert", "--org", username, "--name", "python3-pkgtest")
 	cp.ExpectExitCode(0)
 
 	cp = ts.Spawn("activate", namespace, "--path="+ts.Dirs.Work, "--output=json")
 	cp.ExpectExitCode(0)
 
-	suite.Run("packages add", func() {
-		cp := ts.Spawn("packages", "add", "dateparser@0.7.2")
-		cp.ExpectRe("(?:Package added|The project is currently building)")
-		cp.Wait()
-	})
+	cp = ts.Spawn("history")
+	cp.ExpectExitCode(0)
 
-	suite.Run("packages update", func() {
-		cp := ts.Spawn("packages", "update", "dateparser@0.7.6")
-		cp.ExpectRe("(?:Package updated|The project is currently building)")
-		cp.Wait()
-	})
+	// Get the first commitID we find, which should be the first commit for the project
+	commitRe := regexp.MustCompile(`[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}`)
+	firstCommit := commitRe.FindString(cp.TrimmedSnapshot())
```

Comment:
Went this route for getting the commit to revert to. Created a story here: https://www.pivotaltracker.com/story/show/175487029 to follow up on the output of `state history` and now `state revert`